Qualification tool - Add option to filter by minimum event log size

tgravescs opened the pull request:
Similar to https://github.com/NVIDIA/spark-rapids-tools/pull/1275, bad here filter out files smaller then a certain size.  This allows users to do ranges.  


```
  -m, --min-event-log-size  <arg>    Process only application event logs whose
                                     size is greater than the size specified.
                                     Valid units of size are
                                     b(bytes),k(kilobytes),m(megabytes),g(gigabytes).
                                     If no units are specified, the size is
                                     assumed to be m. Note, this does not
                                     support event log rolling which puts
                                     multiple event logs for the same
                                     application into a single directory.
```


Manually tested.  Tested with this option alone and then also with max size and the filter criteria to make sure all work together.

<!--

Thank you for contributing to Tools of RAPIDS Accelerator for Apache Spark!

Here are some guidelines to help the review process go smoothly.

1. Please write a description in this text box of the changes that are being
   made.

2. Please ensure that you have written units tests for the changes made/features
   added.

3. If you are closing an issue please use one of the automatic closing words as
   noted here: https://help.github.com/articles/closing-issues-using-keywords/

4. If your pull request is not ready for review but you want to make use of the
   continuous integration testing facilities please label it with `[WIP]`.

5. If your pull request is ready to be reviewed without requiring additional
   work on top of it, then remove the `[WIP]` label (if present).

6. Once all work has been done and review has taken place please do not add
   features or make changes out of the scope of those requested by the reviewer
   (doing this just add delays as already reviewed code ends up having to be
   re-reviewed/it is hard to tell what is new etc!). Further, please avoid
   rebasing your branch during the review process, as this causes the context
   of any comments made by reviewers to be lost. If conflicts occur during
   review then they should be resolved by merging into the branch used for
   making the pull request.

Many thanks in advance for your cooperation!

-->

parthosa reviewed (approved):
Thanks @tgravescs. nit: Short flag `-m` might conflict with `max` filter arg.

tgravescs commented:
actually it looks like it changed -m to be min instead of max, let me look at that

tgravescs commented:
fixed the short -m option to go back to --max-event-log-size.  I also changed this min to be >= as I think that makes more sense to user.

```
  -m, --max-event-log-size  <arg>    Process only application event logs whose
                                     size is less than or equal to the size
                                     specified. Valid units of size are
                                     b(bytes),k(kilobytes),m(megabytes),g(gigabytes).
                                     If no units are specified, the size is
                                     assumed to be m. Note, this does not
                                     support event log rolling which puts
                                     multiple event logs for the same
                                     application into a single directory.
      --max-sql-desc-length  <arg>   Maximum length of the SQL description
                                     string output with the per sql output.
                                     Default is 100.
      --min-event-log-size  <arg>    Process only application event logs whose
                                     size is greater than or equal to the size
                                     specified. Valid units of size are
                                     b(bytes),k(kilobytes),m(megabytes),g(gigabytes).
                                     If no units are specified, the size is
                                     assumed to be m. Note, this does not
                                     support event log rolling which puts
                                     multiple event logs for the same
                                     application into a single directory.

```

parthosa reviewed (approved):
Thanks @tgravescs.
